### PR TITLE
Added the cpp standard

### PR DESCRIPTION
**Pull request by prrockzed**
# Description

Allows the cpp file to run on the latest c++ standard, i.e., c++20.
Added the flag '-std=c++20' for smooth functioning of the code in current cpp files.

### CODE REVIEW

**Comment by proffapt** [comment]
Umm, isse baaki versions par farak nahi padega?
Woh compilation errors toh nahi de rahe?

**Comment by prrockzed** [comment]
Nope. C++20 provides support to all its previous version.
Refer to this link for confirmation: https://gcc.gnu.org/onlinedocs/gcc/Standards.html

File: run [hunk 1]
```diff
@@ -83,6 +83,7 @@ run() {
 	# C[++] fam specifics
 	default_debugger
 	CFLAGS="-Wall"
+  CPPSTD="-std=c++20"
```
Comment (proffapt) [inline run]:
```suggestion
    CPPSTD="-std=c++20"
```

Just aligning the text.

Comment (proffapt) [reply run]: Commit this suggestion and it will be good to go :)

**Comment by prrockzed** [comment]
Done.

**Comment by proffapt** [comment]
> Done.

Bhai do baar same commit kardiya, double ident hogaya.. lol

File: run [hunk 2]
```diff
@@ -83,6 +83,7 @@ run() {
 	# C[++] fam specifics
 	default_debugger
 	CFLAGS="-Wall"
+        CPPSTD="-std=c++20"
```
Comment (proffapt) [inline run]:
```suggestion
    CPPSTD="-std=c++20"
```

Just commit this

**Comment by prrockzed** [comment]
Ohh, acha :)

**Comment by prrockzed** [comment]
Hogya

**Review by proffapt — APPROVED** [review]
LGTM

**Pull request merged**